app/server/src/app.ts

```diff
@@ -95,7 +97,13 @@ class App {
     this.app.use(cors({ origin: config.get('apiUrl'), credentials: true }));
 
     this.app.use(hpp());
-    this.app.use(helmet());
+    this.app.use(
```

Comment:
I don't exactly understand why, but the page doesn't load without this security setting in production mode.